datadog/resource_datadog_dashboard.go

```diff
@@ -3850,3 +3852,26 @@ func buildTerraformWidgetAxis(datadogWidgetAxis datadog.WidgetAxis) map[string]i
 	}
 	return terraformWidgetAxis
 }
+
+func validateDashboardLayoutType(val interface{}, key string) (warns []string, errs []error) {
+	value := val.(string)
+	switch value {
+	case "free", "ordered":
+		break
+	default:
+		errs = append(errs, fmt.Errorf(
+			"%q contains an invalid value %q. Valid values are `free` or `ordered`", key, value))
+	}
+	return
+}
+func validateGroupWidgetLayoutType(val interface{}, key string) (warns []string, errs []error) {
+	value := val.(string)
+	switch value {
```

Comment:
Adding a `switch` despite this being one value only because we plan to support more layout types in the future.